sorc/ncep_post.fd/MDL2P.f

```diff
@@ -1259,13 +1261,7 @@ SUBROUTINE MDL2P(iostatusD3D)
               ENDDO
             ENDDO
 !
-            IF(MODELNAME == 'GFS' .or. MODELNAME == 'FV3R')THEN
-              CALL CALRH_GFS(EGRID2(1,jsta),TSL(1,jsta),QSL(1,jsta),EGRID1(1,jsta))
-            ELSEIF (MODELNAME == 'RAPR')THEN 
```

Comment:
@JesseMeng-NOAA Please keep GSD_RH here.

Reply:
Saw your solution for GSD.

Reply:
The CALRH_GSD or CALRH_NAM logical is made in UPP_PHYSICS.f

Reply:
Got it.